src/Hook/Event/Listener.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/**
+ * This file is part of CaptainHook
+ *
+ * (c) Sebastian Feldmann <sf@sebastian-feldmann.info>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace CaptainHook\App\Hook\Event;
+
+use CaptainHook\App\Config;
+use CaptainHook\App\Console\IO;
+use SebastianFeldmann\Git\Repository;
+
+/**
+ * Interface EventListener
+ *
+ * @package CaptainHook
+ * @author  Sebastian Feldmann <sf@sebastian-feldmann.info>
+ * @link    https://github.com/captainhookphp/captainhook
+ * @since   Class available since Release 0.9.0
+ */
+interface Listener
```

Comment:
This should be a `Subscriber` i think.
As it does not actually "listen" to events but does the subscription. Either on its own or pass it on to another handler.